orchestrator/src/types/constant.rs

```diff
@@ -14,7 +14,7 @@ pub const STORAGE_STATE_UPDATE_DIR: &str = "state_update";
 pub const STORAGE_BLOB_DIR: &str = "blob";
 pub const STORAGE_ARTIFACTS_DIR: &str = "artifacts";
 pub const BLOB_LEN: usize = 4096;
-pub const MAX_BLOBS: usize = 6;
+pub const MAX_BLOBS: usize = 6; // TODO: This should be configurable via ENV or config file
```

Comment:
let's follow this practice 🙏 
```suggestion
pub const MAX_BLOBS: usize = 6; // TODO(prakhar 07/02/2025): This should be configurable via ENV or config file
```

Reply:
Okay. have created a PR for this, so should not be a TODO when this PR is merged